compredict/client.py

```diff
@@ -80,8 +81,24 @@ def verify_peer(self, option: str):
     def last_error(self) -> Error:
         return self.connection.last_error
 
+    def set_callback_urls(self, callback_url: Union[list, str]) -> str:
```

Comment:
Add `Union[List[str], str]` to be more precise where `List` is coming from `from typing import List`. Do the same for def run_algorithm.

Reply:
In addition, I think set_callback_urls will only be ran within the class, so we can make it protected, so I suggest to name the function `_set_callback_urls`

Reply:
Done.